group_vars/all/dcos.yaml.example

```diff
@@ -3,7 +3,7 @@ dcos:
   download: "https://downloads.dcos.io/dcos/stable/1.13.4/dcos_generate_config.sh"
   download_checksum: "sha256:55b096236a6de1a40259e6d8a8a21861c5333cbe2b9ba3e274d5de1534b095dc"
   download_win: "https://downloads.dcos.io/dcos/testing/master/windows/dcos_generate_config_win.sh"
-  download_win_checksum: "sha256:edfbb4549a78237a0208d2226333d4ec73abf572f00f3a15bb420b0e3bde9d0c"
+  #download_win_checksum: "sha256:edfbb4549a78237a0208d2226333d4ec73abf572f00f3a15bb420b0e3bde9d0c"
```

Comment:
wait wait... removing the checksum doesn't fix... it should be replaced with the correct checksum

Reply:
@fatz , current `testing` artifacts will be changed every day, until BETA will be prepared for customers. Therefore it doesn't make any sense to hold win_checksum in the example. 
Could you please help in adopting the Molecule tests for EE variant?

Reply:
well this should contain at least a working immutable download so one could use the example otherwise this file wouldn't make any sense.

Instead of master we of course should use a given commit id. Beside that I don't think that the lastest build work with 1.13.4 correct? It should probably be 2.0 

```
curl https://versions.d2iq.com/version/open/2.0.0 | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   240  100   240    0     0    784      0 --:--:-- --:--:-- --:--:--   784
{
  "version": "2.0.0",
  "url": "https://downloads.dcos.io/dcos/stable/2.0.0/dcos_generate_config.sh",
  "sha256": "7b966416c79c1c8887ac765ac45725c749570dfce545b25150e154c6ba270dbd",
  "commit": "dbf9379df9e37a1d341e0d619a2f23439431023b",
  "variant": "open"
}
```

Reply:
@fatz, 
Having no access to the S3 bucket, may I ask you to upload the artifact into stable 2.0 version and provide me with a link? If you have no access either, will contact CI team.
Upon this I will be able to modify checksum. What about EE version - will you be able to adjust the Molecule test?